core/src/main/java/de/danoeh/antennapod/core/storage/PodDBAdapter.java

```diff
@@ -1381,6 +1389,12 @@ public void onCreate(final SQLiteDatabase db) {
         public void onUpgrade(final SQLiteDatabase db, final int oldVersion, final int newVersion) {
             Log.w("DBAdapter", "Upgrading from version " + oldVersion + " to " + newVersion + ".");
             DBUpgrader.upgrade(db, oldVersion, newVersion);
+            if (oldVersion == 1090000) {
```

Comment:
Please move this to `DBUpgrader`. Also, use `oldVersion < 1090001`. Users might skip AntennaPod versions. When checking for equality, this skips the migration when coming from, say `1080100` instead of exactly `1090000`.